src/vsr/replica.zig

```diff
@@ -4564,11 +4569,10 @@ pub fn Replica(
 
             assert(self.commit_min == self.commit_max);
             assert(self.repair_pipeline_op() == null);
+            self.verify_pipeline();
```

Comment:
Can we include a note that `self.verify_pipeline()` is technically redundant here because of the previous `repair_pipeline_op` assertion?